app/models/notifications/mentioned.rb

```diff
@@ -13,19 +13,26 @@ def deleted_translation_key
     end
 
     def linked_object
-      target.post
+      target.mentions_container
     end
 
     def self.notify(mentionable, recipient_user_ids)
       actor = mentionable.author
+      relevant_mentions = filter_mentions(
+        mentionable.find_or_create_mentions.local.where.not(person: mentionable.author),
```

Comment:
That is actually the same thing that we do know when we request `mentioned_people` for StatusMessage (see https://github.com/diaspora/diaspora/blob/develop/app/models/status_message.rb#L61).

However, we won't need this, if we introduce changes we discussed above in https://github.com/diaspora/diaspora/pull/6818#discussion-diff-72363123